queries/vrl/highlights.scm

```diff
@@ -0,0 +1,118 @@
+(comment) @comment
+
+[
+  (null)
+  (boolean)
+] @constant.builtin
+
+(timestamp) @constant
+
+(closure_variables
+  "|" @punctuation.bracket)
```

Comment:
Can you move this after the `@operator` query? Otherwise that one overrides this one